pkg/controller/bundle/bundle_unpacker_test.go

```diff
@@ -488,8 +489,8 @@ func TestConfigMapUnpacker(t *testing.T) {
 										},
 										{
 											Name:            "pull",
-											Image:           bundlePath,
-											ImagePullPolicy: "Always",
+											Image:           digestPath,
+											ImagePullPolicy: "IfNotPresent",
```

Comment:
I think this fails to test that we have the pull policy set to `always` if a bundlePath is provided, is this captured elsewhere?

Reply:
All of the other test cases in this file use a 'normal' (non-digest) bundlePath and expect the resulting pull policy to be `Always`. Because of that I figured we'd only need a single case in here that covers the `IfNotPresent` scenario.

Reply:
That's probably sufficient, in more complex workflows we'd want to ensure that each "type" of image is tested in all places, but I think this one is pretty safe.